File: WebCore/lib/ds/employees/src/lib/search/shared/models/employee-search-filter.ts
import { EmployeeSearchFilterType } from "./employee-search-filter-type";
import { IEmployeeSearchFilterOption } from "./employee-search-filter-option";

export interface IEmployeeSearchFilter {
    filterType:    EmployeeSearchFilterType;
    description:   string;
    $selected?:    IEmployeeSearchFilterOption;
    filterOptions: IEmployeeSearchFilterOption[];
}

File: Ds.Core/Dominion.Core.Dto/Employee/Search/EmployeeSearchFilter.cs
using System.Collections.Generic;

namespace Dominion.Core.Dto.Employee.Search
{
    public class EmployeeSearchFilter
    {
        public EmployeeSearchFilterType                 FilterType    { get; set; }
        public string                                   Description   { get; set; }
        public IEnumerable<IEmployeeSearchFilterOption> FilterOptions { get; set; }
    }
}